projects/igniteui-angular/src/lib/core/styles/components/checkbox/_checkbox-theme.scss

```diff
@@ -222,14 +230,21 @@
 
     %cbx-composite--invalid {
         border-color: var-get($theme, 'error-color');
+
+        @if $variant == 'bootstrap' and $theme-variant == 'dark' {
+            %cbx-composite-mark {
+                stroke: black;
```

Comment:
Why are you using a hardcoded color?

Reply:
same reason, no need to expose new property, since it's different only in dark bootstrap theme